src/backend/api/reviews.js

```diff
@@ -0,0 +1,81 @@
+const express = require("express");
+const router = express.Router();
+const knex = require("../database");
+
+router.get("/", async (req, res) => {
+  try {
+    const reviews = await knex("review");
+    res.json(reviews);
+  } catch (err) {
+    throw new Error("something went wrong on fetching reviews");
+  }
+});
+router.post("/", async (req, res) => {
+  try {
+    const reviewToAdd = {
+      title: req.body.title,
+      description: req.body.description,
+      stars: req.body.stars,
+      meal_id: req.body.meal_id,
+      created_date: req.body.created_date,
+    };
+    const result = await knex("review").insert(reviewToAdd);
+    res.json(result);
+  } catch (error) {
+    throw new Error("something went wrong with adding meal");
```

Comment:
Same there here the error message is not explaining the correct thing

Reply:
thanks